flink-container/docker/build.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/sh
```

Comment:
Just want to note that in other scripts, `bash` is used. Also `/usr/bin/env` is used to determine the `bash` binary. See `config.sh` for an example.

Reply:
Good point. I will correct it.